src/utils/morello-api.js

```diff
@@ -1,9 +1,14 @@
 import qs from 'qs'
 import { DSBD_API_URL } from './env'
 
-export function executeBinary(name, params) {
+export async function executeBinary(name, params) {
   const queryString = qs.stringify(params)
   const url = `${DSBD_API_URL}/scenario/${name}`
 
-  return fetch(`${url}?${queryString}`)
+  try {
```

Comment:
Not sure if much use of this here....

Reply:
I'd probably say no this isn't much use. This code doesn't really know how to handle the error and it's not doing any data transformations on the error that are useful. It's probably better to let the async rejection propagate up the call stack. That said not a biggy